tests/test_resources.py

```diff
@@ -3393,6 +3393,49 @@ def test_account_external_accounts(self):
         with self.mock_request('account/created-with-external-accounts.xml'):
             account.save()
 
+    def test_list_general_ledger_accounts(self):
+        with self.mock_request('general_ledger_accounts/list.xml'):
+            general_ledger_accounts = GeneralLedgerAccount.all()[0]
+
+        self.assertEqual(general_ledger_accounts.id, 'u90r5deeaxix')
+        self.assertEqual(general_ledger_accounts.code, 'code1')
+        self.assertEqual(general_ledger_accounts.account_type, 'revenue')
+        self.assertEqual(general_ledger_accounts.description, 'Some Description')
```

Comment:
I think for an index endpoint, we want to test if we got many objects in return, also would be nice to have a test using the filter by account_type.

Reply:
updated test to check for count instead 👍